build.gradle

```diff
@@ -185,6 +196,10 @@ test {
     dependsOn buildJniLib
     systemProperty 'tests.security.manager', 'false'
     systemProperty "java.library.path", "$rootDir/jni/release"
+    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+        // Add the paths of built JNI libraries and its dependent libraries to PATH variable in System variables
+        environment('PATH', System.getenv('PATH') + ";$rootDir/jni/release" + ";$rootDir/src/main/resources/windowsDependencies")
```

Comment:
Seems like this can be added to windowsScript.ps1?
So you dont have to call this multiple times?

Reply:
As discussed we can't set those paths in script. because it gets wiped off later. Leaving it as it is and added conditions to avoid adding to path variable multiple times

Reply:
After checking with @naveentatikonda we figured out this is a temp export of the variable, thus wont preserve after shell getting closed, therefore no need to check prior existence of the value.

temporary: environment(<varname>, <value>)
permanent: environment(<varname>, <value>, scope)

Therefore no need to make changes here.